### PR TITLE
file organisation for tests

### LINKED ISSUE #2159: Clean up `// TODO` and `cy.wait` in tests
**Description**
In tests we have a lot of `// TODO` and `cy.wait` in tests code. For example some of it are provided because Configuration is out of DOM and we have to figure out why and fix that.

**Expected result**
Tests don't have `// TODO` and `cy.wait`

**Actual result**

**Steps to reproduce**

**Troubleshooting**

### PR DESCRIPTION
**Pull request by mrCherry97**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**
part of https://github.com/kyma-project/busola/issues/2150
Closes #2159

### CODE REVIEW

**Comment by mrCherry97** [comment]
image
cluster: https://status.build.kyma-project.io/view/gs/kyma-prow-logs/pr-logs/pull/kyma-project_busola/2260/mrcherry97_test_of_prowjob_pre-busola-integration-test-k3s-clus/1612367767858581504
namespace: https://status.build.kyma-project.io/view/gs/kyma-prow-logs/pr-logs/pull/kyma-project_busola/2260/mrcherry97_test_of_prowjob_pre-busola-integration-test-k3s-name/1612367788393893888
lighthouse: https://status.build.kyma-project.io/view/gs/kyma-prow-logs/pr-logs/pull/kyma-project_busola/2260/mrcherry97_test_of_prowjob_pre-busola-lighthouse/1612367761571319808
local: https://status.build.kyma-project.io/view/gs/kyma-prow-logs/pr-logs/pull/kyma-project_busola/2260/mrcherry97_test_of_prowjob_pre-busola-local/1612367851291676672
docker: https://status.build.kyma-project.io/view/gs/kyma-prow-logs/pr-logs/pull/kyma-project_busola/2260/mrcherry97_test_of_prowjob_pre-busola-smoke-test-docker/1612367807486365696
web: https://status.build.kyma-project.io/view/gs/kyma-prow-logs/pr-logs/pull/kyma-project_busola/2260/mrcherry97_test_of_prowjob_pre-busola-web/1612367826843078656

**Review by grego952 — DISMISSED** [review]

File: tests/integration/tests/cluster/test-cluster-overview.spec.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ context('Test Cluster Overview', () => {
   });
 
   it('Go to Node details', () => {
-    cy.wait(500); // TODO
+    cy.wait(500);
```
Comment (Lyczeq) [inline tests/integration/tests/cluster/test-cluster-overview.spec.js:32]: Can't we remove this `cy.wait`?

File: tests/integration/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
+# Tests
+
+## Overview
+
+This project contains smoke and integration UI tests for Busola.
+
+## Prerequisites
+
+Before testing, you need to copy your cluster's kubeconfig file to `fixtures/kubeconfig.yaml`.
```
Comment (Lyczeq) [inline tests/integration/README.md:9]: One small comment regarding this path. In the lighthouse's README there is written `Make sure your kubeconfig is copied as 'lighthouse/fixtures/kubeconfig.yaml'`. In the integration README, there is not `integration/` prefix.  Please choose one option for both paths, with or without the prefix.

**Review by Lyczeq — APPROVED** [review]

**Review by grego952 — APPROVED** [review]

**Pull request merged**